tests/integration/test_config.py

```diff
@@ -339,3 +378,381 @@ async def test_host_group_config(cluster: Cluster) -> None:
     param: ParameterHG = config_1["more"]["strange"]  # type: ignore
     # bit of ADCM logic: sync parameter shipped from object's config
     assert param.value == strange_val_1
+
+
+async def test_config_two_sessions(
+    adcm: ADCMContainer, httpx_client: AsyncClient, cluster: Cluster, simple_hostprovider_bundle: Bundle
+) -> None:
+    kwargs = {"verify": False, "timeout": 10, "retry_interval": 1, "retry_attempts": 1}
+    credentials = Credentials(username="admin", password="admin")  # noqa: S106
+
+    async with ADCMSession(url=adcm.url, credentials=credentials, **kwargs) as second_client:
```

Comment:
for these tests session preparation specifics are unimportant

you may hide it in func or even add second user session fixture

see comment in similar @Starovoitov PR

Reply:
refactored